src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -677,7 +677,7 @@ constexpr uint32_t Limitations::kBytesPerCropElement;
 constexpr uint32_t Limitations::kBytesPerConcatElement;
 constexpr uint32_t Limitations::kMemoryPageSize;
 
-thread_local std::shared_ptr<Limitations> Limitations::k_instance{nullptr};
+InferenceEngine::ThreadLocal<std::shared_ptr<Limitations>> Limitations::kInstance{nullptr};
```

Comment:
Please use `ov::threading::ThreadLocal` instead of `InferenceEngine::ThreadLocal`

Reply:
ok